src/table.rs

```diff
@@ -26,6 +27,7 @@ impl<T, U> Table<T, U> {
         }
     }
 
+    /// Create a column with this table.
```

Comment:
Adding some example here would be great. Additionally we should document here what the `ST` generic type stands for because the user is expected to provide some concrete type here.

Reply:
Good idea. I'm glad to add more examples as I learn more. My ideal is to publish what's ready now, so the crate can go to crates.io, and I'll add more examples in a later PR when I know more about what works well.

Reply:
Is it ok to mark "Resolve" for this PR comment?